docs/hardware.rst

```diff
@@ -322,26 +324,25 @@ touch the Internet or run an operating system other than Tails, it does
 not need a hard drive or network device; in fact, we recommend removing
 these components if they are already present.
 
-One option is to buy a Linux-compatible laptop such as a `Lenovo
-Thinkpad <http://shop.lenovo.com/us/en/laptops/thinkpad/t-series/t540p/>`__.
-You can also repurpose an old laptop if you have one available.
-
-Another option is to buy an `Intel NUC
-D54250WYK <http://www.amazon.com/Intel-D54250WYK-DisplayPort-Graphics-i5-4250U/dp/B00F3F38O2/>`__
-(same model as the servers) with a power cord and `4 GB of
-memory <http://www.amazon.com/Crucial-PC3-12800-204-Pin-Notebook-CT2CP25664BF160B/dp/B005MWQ6WC/>`__,
-but note that you will also need to get a monitor and a wired keyboard
-and mouse. It does not come with a hard drive or wireless networking
-hardware by default, so you will not need to remove these components
-before using it. However, we do recommend taping over the IR receiver
-with some opaque masking tape.
-
-Note that if you do want to use a NUC for the *SVS*, you *should not* use
-any of the new generation of NUCs, which have names starting with "NUC5"
-(e.g.
-`NUC5i5RYK <https://www-ssl.intel.com/content/www/us/en/nuc/nuc-kit-nuc5i5ryk.html>`__..
-These NUCs have wireless networking built into the motherboard, and it
-is impossible to physically remove.
+One option is to buy a Linux-compatible laptop such as a
+`Lenovo ThinkPad <http://www3.lenovo.com/us/en/laptops/thinkpad/thinkpad-t-series/c/thinkpadt>`__;
```

Comment:
That could be debated, sure; I simply replaced the prior recommendation, T540p, with the T420, since we recently tested this one in-house.